app/assets/stylesheets/ama_layout/layout_components/tables.scss

```diff
@@ -86,6 +102,11 @@
   }
 }
 
+.table--ash-bg{
```

Comment:
there is already an --ash-bg modifier on the .table class above in this file, so i think you can remove this